packages/web/src/pages/history-page/components/desktop/HistoryPage.tsx

```diff
@@ -22,79 +26,77 @@ import styles from './HistoryPage.module.css'
 export type HistoryPageProps = {
   title: string
   description: string
-  userId: ID
-  entries: any
-  dataSource: any
-  playingIndex: number
-  isEmpty: boolean
-  loading: boolean
-  queuedAndPlaying: boolean
-  onClickRow: (record: any) => void
-  onClickSave: (record: any) => void
-  onClickTrackName: (record: any) => void
-  onClickArtistName: (record: any) => void
-  onClickRepost: (record: any) => void
-  onSortTracks: (sorters: any) => void
-  goToRoute: (route: string) => void
-  onPlay: () => void
-  onFilterChange: (e: ChangeEvent<HTMLInputElement>) => void
-  filterText: string
-  fetchNextPage: () => void
-  isFetchingNextPage: boolean
-  totalRowCount?: number
-  pageSize?: number
 }
 
-const HistoryPage = ({
-  title,
-  description,
-  userId,
-  dataSource,
-  playingIndex,
-  isEmpty,
-  loading,
-  queuedAndPlaying,
-  onClickRow,
-  onClickSave,
-  onClickTrackName,
-  onClickArtistName,
-  onClickRepost,
-  onSortTracks,
-  goToRoute,
-  onPlay,
-  onFilterChange,
-  filterText,
-  fetchNextPage,
-  isFetchingNextPage,
-  totalRowCount,
-  pageSize
-}: HistoryPageProps) => {
+const pageSize = 50
```

Comment:
nit: caps case for a constant
```suggestion
const PAGE_SIZE = 50
```